app.py

```diff
@@ -98,6 +112,18 @@ def update_cve_info() -> None:
                         advisory=cve["advisory"],
                     )
 
+                    publish_to_topic(
```

Comment:
@pacospace @fridex  I chose this place to send messages after CVE record is created in the database, so is it the right place WDYT?

Reply:
Sounds good :+1: